compiler/next/test/resolution/testResolve.cpp

```diff
@@ -94,47 +99,68 @@ static void test2() {
     std::string contents = "var x;";
     setFileText(context, path, contents);
 
-    auto& vec = resolveFile(context, path);
+    const ModuleVec& vec = parse(context, path);
+    assert(vec.size() == 1);
+    const Module* m = vec[0]->toModule();
+    assert(m);
+    resolveModule(context, m->id());
+
+    context->collectGarbage();
+  }
+
+  {
+    printf("part 3\n");
+    context->advanceToNextRevision(true);
+    auto path = UniqueString::build(context, "input.chpl");
+    std::string contents = "var x: int;";
+    setFileText(context, path, contents);
+
+    const ModuleVec& vec = parse(context, path);
     assert(vec.size() == 1);
-    const Module* module = vec[0]->decl->toModule();
-    ASTNode::dump(module, 2);
-    const Variable* varDecl = module->child(0)->toVariable();
-    assert(varDecl);
-    assert(0 == varDecl->name().compare("x"));
+    const Module* m = vec[0]->toModule();
+    assert(m);
+
+    const Variable* x = m->stmt(0)->toVariable();
+    assert(x);
+
+    const ResolutionResultByPostorderID& rr = resolveModule(context, m->id());
+    assert(rr.byAst(x).type.type()->isIntType());
+
     context->collectGarbage();
   }
 
+
   for (int i = 0; i < 3; i++) {
```

Comment:
Are you doing this multiple times to demonstrate that you don't get a different result when resolving multiple times? You may want to add a comment explaining the intent.